schema/dataKeys.json

```diff
@@ -437,6 +437,54 @@
                   }
                 }
               },
+              "productInfo": {
+                "type": "object",
+                "properties": {
+                  "productName": {
+                    "type": "string",
+                    "description": "productName in content gateway"
+                  },
+                  "productCode": {
+                    "type": "string",
+                    "description": "productCode in content gateway"
+                  },
+                  "productVersionName": {
+                    "type": "string",
+                    "description": "productVersionName in content gateway"
+                  }
+              },
+              "starmap": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "type": "string",
+                      "description": "Artifact name"
+                    },
+                    "workflow": {
+                      "type": "string",
+                      "description": "Push workflow"
+                    },
+                    "cloud": {
+                      "type": "string",
+                      "description": "Cloud provider's name"
+                    },
+                    "mappings": {
+                      "type": "object",
+                      "description": "Mappings for the given artifact"
+                    },
+                    "billing-code-config": {
+                      "type": "object",
+                      "description": "Billing configuration for the community worklow"
+                    }
+                  }
+                }
+              },
+              "cloudMarketplacesSecret": {
+                "type": "string",
+                "name": "Secret for cloud marketplaces"
```

Comment:
This should be description, not name, right?

Reply:
oops

Reply:
good catch! Changed that